org.eclipse.scanning.test/src/org/eclipse/scanning/test/scan/nexus/MonitorTest.java

```diff
@@ -278,7 +281,7 @@ private void assertPerScanMonitors(IRunnableDevice<ScanModel> scanner, List<Stri
 		NXentry entry = rootNode.getEntry();
 		NXinstrument instrument = entry.getInstrument();
 
-		Collection<IScannable<?>> perScan  = scanner.getModel().getMonitors().stream().filter(scannable -> scannable.getMonitorRole()==MonitorRole.PER_SCAN).collect(Collectors.toList());
+		Collection<IScannable<?>> perScan  = scanner.getModel().getMonitorsPerScan();
 
 		// check each metadata scannable has been written correctly
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)